KeyboardVisualizerCommon/LEDStrip.cpp

```diff
@@ -275,16 +293,16 @@ void LEDStrip::SetLEDsHuePlus(COLORREF pixels[64][256])
     {
         unsigned char *serial_buf;
 
-        serial_buf = new unsigned char[250];    //Size of Message always 5 XX Blocks (Mode Selection) +  3 XX for each LED (1 color) 
+        serial_buf = new unsigned char[125];    //Size of Message always 5 XX Blocks (Mode Selection) +  3 XX for each LED (1 color) 
```

Comment:
I see you're repeating 125 (previously 250) multiple times, maybe it's worth to create a const for that?